src/style.css

```diff
@@ -1 +1,26 @@
 /* add styles here */
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
```

Comment:
Ensure that the `star.svg` file exists in the `images` directory to avoid broken image links.